src/backend/aspen/api/utils/sample.py

```diff
@@ -243,12 +244,48 @@ def format_sample_lineage(sample: Sample) -> List[Dict[str, Any]]:
             lineage_response["scorpio_support"] = lin.raw_lineage_output.get(
                 "scorpio_support"
             )
+            # For `last_updated`, only SC2 (via Pangolin) provides a non-null val.
+            lineage_response["last_updated"] = lin.last_updated
+        else:
+            # Currently (Dec 2022), other pathogens get lineage via Nextclade.
+            # In Nextclade, dataset `tag` is a datetime, so pull from that.
+            lineage_response["last_updated"] = get_date_from_nextclade_tag(
+                lin.reference_dataset_tag
+            )
 
         lineages.append(lineage_response)
 
     return lineages
 
 
+def get_date_from_nextclade_tag(reference_dataset_tag: str) -> Optional[str]:
+    """Pulls out "YYYY-MM-DD" date from a Nextclade dataset's `tag` attribute.
+
+    Nextclade attaches a `tag` to every dataset to uniquely identify it. This
+    tag is consistently just a datetime (eg, "2021-06-25T00:00:00Z"), so we
+    can extract the date from it. While Nextclade seems to strongly adhere to
+    this convention for tags, there is no documentation that future tags /must/
+    keep this structure. If they ever change it we'll need to alter how we
+    handle capturing the date associated with a given dataset.
+    """
+    date = None  # default fall-through just in case tag is missing
+    if reference_dataset_tag:
+        m = re.match(r"^\d{4}-\d+-\d+(?=T)", reference_dataset_tag)
```

Comment:
personally I'd use [arrow](https://arrow.readthedocs.io/en/latest/) here instead of a regex, but this should get the job done if nextclade's data is pretty clean

Reply:
I thought about that, but I'm currently kinda thinking of this as a canary in a coal mine: if anything about how they structure their `tag` values changes, I want to find out quickly, because it might indicate some other stuff having changed that we need to look into.